src/js/containers/LoginPage/LoginPage.js

```diff
@@ -17,19 +37,126 @@ class LoginPage extends Component {
         }
     }
 
+    openLinkInWindow(link) {
+        window.open(link, '_blank', 'toolbar=0,status=0,width=720,height=700');
+    }
+
+    handleEmailChange(email) {
+        this.setState({ email });
+    }
+
+    handleApiKeyChange(apiKey) {
+        this.setState({ apiKey });
+    }
+
+    handleLoginSubmit(e) {
+        e.preventDefault();
+        const { dispatch, config } = this.props;
+
+        let isHostAPILogin = getConfigValue(config, 'useHostAPILogin');
```

Comment:
Thats not true.  `const` for values that won't change, `let` for anywhere you would be using `var` since its more tightly scoped.  Good call Furkan - lets continue to refactor these as we go to be more consistent.